operation/manifest.go

```diff
@@ -81,6 +83,49 @@ type AppManifestService struct {
 	Parameters  map[string]interface{} `yaml:"parameters,omitempty"`
 }
 
+func (ams *AppManifestService) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var raw interface{}
+	if err := unmarshal(&raw); err != nil {
+		return err
+	}
+	switch v := raw.(type) {
+	case string:
+		ams.Name = v
+	case map[interface{}]interface{}:
```

Comment:
It can never be map[interface{}]interface{} type. 
Please let me know if I am missing something if not do remove it.

Reply:
It seems like you are utilizing the yaml.v3 library, whereas yaml.v2 requires the use of map[interface{}]interface{}.

Reply:
Thanks for the info!